<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OAuth2</class>
 <widget class="QDialog" name="OAuth2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="wizard">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="step1">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Auth end point</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="endpoint"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Client ID</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="clientId1"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Client secret</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="clientSecret1"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Redirect URI</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="redirectUri1">
         <property name="text">
          <string>localhost:&lt;random&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Scope</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="scope"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>State</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="state"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Response type</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="respType">
         <property name="text">
          <string>code</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="cancelbtn1">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startBtn">
           <property name="text">
            <string>Start</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="step2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>OAuth server</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWebView" name="webView">
         <property name="url">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="8" column="0" colspan="5">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="cancelBtn2">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextBtn">
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Code</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="resultcode"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Client ID</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Redirect URI</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Grant type</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="clientId2"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="clientSecret2"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="redirectUri2">
         <property name="text">
          <string>localhost:&lt;random&gt;</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="grantType"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Client secret</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="endpoint2"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Token end point</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Log</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="log">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>clientId1</sender>
   <signal>textChanged(QString)</signal>
   <receiver>clientId2</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clientSecret1</sender>
   <signal>textChanged(QString)</signal>
   <receiver>clientSecret2</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>webView</sender>
   <signal>loadProgress(int)</signal>
   <receiver>progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redirectUri1</sender>
   <signal>textChanged(QString)</signal>
   <receiver>redirectUri2</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
